Lib/multiprocessing/connection.py

```diff
@@ -272,11 +274,24 @@ class PipeConnection(_ConnectionBase):
         """
         _got_empty_message = False
 
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+            self._send_ov = None
```

Comment:
You can simply set it as a class variable, like `_handle` and `_got_empty_message`.

Is is even safer, because `_close()` can be called in `__del__` if `__init__` was interrupted after setting `_handle` and before setting `_send_ov`.

Reply:
Done, thanks for the suggestion.